prow/cmd/tide/README.md

```diff
@@ -1,180 +1,30 @@
 # Tide
 
 Tide is a [Prow](https://github.com/kubernetes/test-infra/blob/master/prow/README.md)
-component for managing a pool of PRs that match a given set of criteria.
-It will automatically retest and merge PRs in the pool if they pass tests
-against the latest base branch commit.
+component for managing a pool of GitHub PRs that match a given set of criteria.
+It will automatically retest and merge PRs in the pool if they have passing and
```

Comment:
Right now this reads like `tide` both retests and merges if the PRs are passing. Maybe something like:

> It will automatically re-test PRs that are ready to merge and automatically merge them when they have passing up-to-date tests.

Reply:
Fixed.